ppr-ui/tests/unit/Dashboard.spec.ts

```diff
@@ -488,9 +491,9 @@ describe('Dashboard add registration tests', () => {
     const getStub = sandbox.stub(axios, 'get')
     const getSearchHistory = getStub.withArgs('search-history')
     getSearchHistory.returns(new Promise(resolve => resolve({ data: { searches: [] } })))
-    const getMyRegDrafts = getStub.withArgs('drafts')
+    const getMyRegDrafts = getStub.withArgs('drafts?fromUI=true&sortCriteriaName=createDateTime&sortDirection=desc')
```

Comment:
Drafts and registrations use the same parameter set, please use startDateTime instead of createDateTime. Same for below.